### PR TITLE
Remove WINDOWS and MSC code

### PR DESCRIPTION
**Pull request by palladia**

### CODE REVIEW

File: Unix/base/user.c [hunk 1]
```diff
@@ -41,84 +40,6 @@
...
-{
-    MI_UNUSED(uid);
-    MI_UNUSED(size);
-    MI_UNUSED(path);
-    MI_UNUSED(content);
-
-    return -1;
-}
-
-#elif defined(CONFIG_POSIX)
```
Comment (paulcallen) [inline Unix/base/user.c]: based on our comment should this #elif not be a $if now?

Comment (paulcallen) [inline Unix/base/user.c]: and where is the #endif?

File: Unix/codec/mof/parser/utility.c [hunk 2]
```diff
@@ -1092,13 +1023,8 @@ void ReportFinalError(
     switch (id)
     {
     case ID_OUT_OF_MEMORY:
-#if defined(_MSC_VER)
-        errorCode = (MI_Uint32)E_OUTOFMEMORY;
-        errorType = MI_RESULT_TYPE_HRESULT;
-#else
         errorCode = ENOMEM;
         errorType = MI_T("STDC"); /* todo: Fixup error type for non-win */
```
Comment (paulcallen) [inline Unix/codec/mof/parser/utility.c]: um, I believe this should be MI_RESULT_TYPE_ERRNO

File: Unix/base/user.c (hunk 1 above)
Comment (palladia) [reply Unix/base/user.c]: I believe this is ok.  No change.

Comment (palladia) [reply Unix/base/user.c]: Also ok.

File: Unix/codec/mof/parser/utility.c (hunk 2 above)
Comment (palladia) [reply Unix/codec/mof/parser/utility.c]: Changed.

**Pull request merged**